### PR TITLE
Truing up a few data conversions

### PR DESCRIPTION
**Pull request by jduv**
**Describe what the PR does:**

Fixes a data type issue where rain data was coming over incorrectly.

**Does this fix a specific issue?**

https://github.com/bachya/ecowitt2mqtt/issues/45
  
**Checklist:**

- [x] Update `README.md` with any new documentation.
- [ ] Add yourself to `AUTHORS.md`.

(Note there is no Authors.md)

### CODE REVIEW

**Comment by jduv** [comment]
Screen Shot 2021-02-07 at 11 34 17 PM

File: docker-compose.dev.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ services:
     build: .
     container_name: ecowitt2mqtt
     environment:
-      LOG_LEVEL: INFO
+      LOG_LEVEL: DEBUG
```
Comment (bachya) [inline docker-compose.dev.yml:10]: Assume this is leftover from your testing – please make sure to revert.

File: docker-compose.dev.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@ services:
       MQTT_PASSWORD: password
       MQTT_USERNAME: ecowitt
       MQTT_TOPIC: "Test"
-      UNIT_SYSTEM: "metric"
+      UNIT_SYSTEM: "imperial"
```
Comment (bachya) [inline docker-compose.dev.yml:19]: Assume this is leftover from your testing – please make sure to revert.

**Review by bachya — COMMENTED** [review]
Just some small comments. Also need to make sure CI passes. Thanks!

**Comment by bachya** [comment]
@jduv Ignore my comment about CI; it's breaking for another reason. Address the comments above and we can merge. Thanks!

**Comment by jduv** [comment]
@bachya 10-4, reverted through another commit. That was my mad, sorry about that. `git commit -am` fail. I figured the CI was sideways since the `pre-commit` didn't run on my commit--I had to turn that off to even get a commit up.

**Pull request merged**